src/enrichment_facility.h

```diff
@@ -233,81 +265,127 @@ class EnrichmentFacility : public cyclus::Facility {
 
   inline double CurrentSwuCapacity() const { return current_swu_capacity; }
 
+  inline void MaxEnrich(double enrichment) { max_enrich = enrichment; }
```

Comment:
it looks like there are a bunch of gcsg function name issues. can you address them each please and ping when ready?